### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by montogeek** [skipjack/react-banner#13]
Hi Skip!

There was a contribution on webpack website regarding accessibility which added a lot of aria attributes to elements, when merging `master` into new `rebuild` branch I find myself unable to add those attributes since `react-banner` doesn't expose a way to set them.

Would you accept a PR to add those attributes as props with sensible defaults?

Edit: PR https://github.com/webpack/webpack.js.org/pull/2294

**Comment by skipjack** [comment]
Yep, will do thanks for reporting this.

**Comment by skipjack** [comment]
Whoops, read that wrong. Yeah, if you'd like to submit a PR that'd be great. Honestly some of those don't even need to be props, I think they can just be set permanently (e.g. `role="search"`). I'd like to keep things as simple as possible but if you think there's some that are worth allowing customization for that's fine.
